src/main/interface/tramp_protocol.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string.h>
+#include "platform.h"
+#include "common/utils.h"
+#include "interface/tramp_protocol.h"
+
+#define TRAMP_SYNC_START            0x0F
+#define TRAMP_SYNC_STOP             0x00
+#define TRAMP_COMMAND_SET_FREQ      'F' // 0x46
+#define TRAMP_COMMAND_SET_POWER     'P' // 0x50
+#define TRAMP_COMMAND_ACTIVE_STATE  'I' // 0x49
+#define TRAMP_COMMAND_GET_CONFIG    'v' // 0x76
+
```

Comment:
My preference is to have all definitions of magic numbers in the header file. You have some here and some there.
I would move all those six defs to `tramp_protocol.h`

Reply:
I think the criterion is if the definitions are intended to be used outside of the `.c` file. The header file is the published interface of the `.c` file, and should not contain anything that isn't meant to be used outside of the `.c` file.

Reply:
Yeah, that is the private/public view of things. At the same time those defines are part of the Tramp Spec and I would like to have the whole spec defined in one place, the header file. Not split in many places. But this is a matter of opinion, style and taste.  No big deal.

Reply:
Just playing devil's advocate... Why would these defines be relevant outside of the interface?

Reply:
Fair enough :) I'm impartial, really.  I guess it could also be built entirely private, including the structs, if a buffer pointer  were passed instead of an instance of `trampFrame_t`

Reply:
Yes, that should be a more generic device independant way. A `trampFrame_t` buffer is very Tramp specific (and possibly private).